cli/src/pcluster/config/cluster_config.py

```diff
@@ -1433,6 +1463,14 @@ def instance_profile(self):
         """Return the IAM instance profile for compute nodes, if set."""
         return self.iam.instance_profile if self.iam else None
 
+    @property
+    def queue_ami_id(self):
```

Comment:
let's rename it to queue_ami to be aligned with custom_ami or official_ami params.

Reply:
done